assets/express/src/database/models/user.model.ts

```diff
@@ -0,0 +1,45 @@
+import { UserRepository } from '../interfaces';
+import { User, UserPayload } from '@entities';
+
+// This is just sample implementation at this point, the DB functionality
+// will be added with next tickets
+// eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-unused-vars
+export default function userOperations(dbConnection: any): UserRepository {
+  async function getUser(email: string): Promise<User> {
+    // Some DB operations...
+
+    return dbConnection.users.get(email);
+  }
+
+  async function getAllUsers(): Promise<User[]> {
```

Comment:
Since this is a repository, IMO we don't need to prefix or suffix it. Just getAll(), create(), delete(), get() is enough.